packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -644,6 +650,144 @@ it('should filter table', () => {
   );
 });
 
+it('should filter false values', () => {
+  const onFilter = jest.fn();
```

Comment:
You are not checking whether it was called and with what value.

Reply:
Nope. I'm checking that we have the desired effect of filtering the table, rather than the specific implementation details.

Reply:
Then it is not worth it to have this variable